utils/parsing.go

```diff
@@ -312,3 +312,26 @@ func GetUniqueCohortDefinitionIdsList(cohortDefinitionIds []int, filterCohortPai
 	uniqueIdsList := MakeUnique(idsList)
 	return uniqueIdsList
 }
+
+func Intersect(list1 []int, list2 []int) []int {
+	result := []int{}
+	for _, list1Item := range list1 {
+		for _, list2Item := range list2 {
```

Comment:
Not sure how large the list can get in the future, maybe converting list2 to set and do look up can be faster for large datasets.

Reply:
these will be relatively short lists, where one of them will be mostly < 100 long and the other one probably not longer than a few thousand entries max.